packages/volto/src/components/theme/RelatedItems/RelatedItems.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * RelatedItems component.
+ * @module components/theme/RelatedItems/RelatedItems
+ */
+
+import { UniversalLink } from '@plone/volto/components';
+import { defineMessages, injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Container } from 'semantic-ui-react';
+
+const messages = defineMessages({
+  relatedItems: {
+    id: 'Related Items',
+    defaultMessage: 'Related Items',
+  },
+});
+
+/**
+ * Related Items component.
+ * @function RelatedItems
+ * @param {array} relatedItems Array of related items.
+ * @returns {JSX.Element} Markup of the component.
+ */
+const RelatedItems = ({ relatedItems, intl }) => {
+  if (!relatedItems || relatedItems.length === 0) {
+    return null;
+  }
+
+  return (
+    <Container style={{ 'margin-top': '20px' }}>
```

Comment:
@wesleybl please don't use inline objects to set css, put a class on it and apply the style from css. If someone needs more space they will need to add an important rule to override the inline style.

Reply:
Done.